pkg/server/wire.go

```diff
@@ -37,6 +37,7 @@ import (
 	"github.com/grafana/grafana/pkg/middleware/loggermw"
 	apiregistry "github.com/grafana/grafana/pkg/registry/apis"
 	provisioningauth "github.com/grafana/grafana/pkg/registry/apis/provisioning/auth"
+	provisioninglegacy "github.com/grafana/grafana/pkg/registry/apis/provisioning/jobs/legacy"
```

Comment:
I know the words have different meaning but legacy is usually interpreted as `deprecated`. What was your intent on calling it legacy?

Reply:
Ideally we can get everything from unified storage/target architecture.  This package uses all the internal raw commands and will not work (easily) in a multi-tenant world 😬 

intention is to make sure it is clear that the stuff in that folder should be replaced (although not yet really possible)